accounts-db/src/accounts_index/in_mem_accounts_index.rs

```diff
@@ -25,6 +25,37 @@ use {
     },
 };
 
+#[derive(Clone, Copy)]
+struct CandidateSelection {
```

Comment:
1. Can we also derive Debug too, please?

2. I'm mostly anti deriving Copy for anything over 8 bytes. Do we need Copy here?